apps/ledger-live-desktop/src/renderer/reducers/dynamicContent.ts

```diff
@@ -61,8 +62,19 @@ export const portfolioContentCardSelector = (state: { dynamicContent: DynamicCon
 export const actionContentCardSelector = (state: { dynamicContent: DynamicContentState }) =>
   state.dynamicContent.actionCards;
 
-export const notificationsContentCardSelector = (state: { dynamicContent: DynamicContentState }) =>
-  state.dynamicContent.notificationsCards;
+export const notificationsContentCardSelector = (state: {
+  dynamicContent: DynamicContentState;
+  settings: SettingsState;
+}) => {
+  const { settings, dynamicContent } = state;
+  return dynamicContent.notificationsCards.map(n => ({
+    ...n,
+    viewed:
+      settings.shareAnalytics || settings.sharePersonalizedRecommandations
```

Comment:
You can use directly trackingEnabledSelector it's the same check. 
I tested it and works well !